app/schema.graphql

```diff
@@ -506,6 +506,11 @@ type Span {
   output: SpanIOValue
   events: [SpanEvent!]!
 
+  """
+  Cumulative token count from self and all descendant spans (children, grandchildren, etc.)
+  """
+  cumulativeTokenCountTotal: Int
```

Comment:
```suggestion
  cumulativeTokenCount: Int
```

I wonder as a follow-up we might want to still break it down by input tokens vs output tokens
<img width="244" alt="Screenshot 2023-08-23 at 11 43 41 AM" src="https://github.com/Arize-ai/phoenix/assets/5640648/3578dc95-8c9d-424d-a8a5-ea8b81d702d5">

Not blocking